drivers/hmis/app/models/hmis/hud/custom_case_note.rb

```diff
@@ -45,6 +47,8 @@ class Hmis::Hud::CustomCaseNote < Hmis::Hud::Base
     where(id: viewable_scope.select(:id))
   end
 
+  scope :ordered_by_date, -> { where.not(information_date: nil).order(information_date: :asc) }
```

Comment:
This is a bit of a  sneaky scope name, since it sounds like it's just sorting but it actually filters out case notes without dates. Maybe clarify the name?